### PR TITLE
Instructions to install bzip2

### PR DESCRIPTION
**Pull request by unreal4u**
bzip2 is not default installed on CentOS systems, so provide general instructions to do this always, even for other systems.

### CODE REVIEW

**Comment by unreal4u** [comment]
If you want, I can also provide CentOS instructions, they aren't that different from the Debian ones, mostly package name changes.

**Comment by mwiencek** [comment]
Thanks. :+1: I think we'd be happy to accept CentOS instructions if the changes are as minimal as they sound, since we've had plenty of people try to set it up there.

File: INSTALL.md [hunk 1]
```diff
@@ -326,7 +330,7 @@ Creating the database
     database level:
 
         ALTER DATABASE musicbrainz_db SET statement_timeout TO 60000;
-
+
```
Comment (mwiencek) [inline INSTALL.md]: Can you remove the unrelated whitespace added there?

**Comment by unreal4u** [comment]
Oh sorry, didn't see that one, removed :)

**Comment by unreal4u** [comment]
Thanks!
